poms/bom-test/pom.xml

```diff
@@ -81,9 +81,9 @@
                 <version>${quarkiverse-micrometer.version}</version>
             </dependency>
             <dependency>
-                <groupId>me.escoffier.certs</groupId>
-                <artifactId>certificate-generator-junit5</artifactId>
-                <version>${certificate.generator.version}</version>
+                <groupId>io.smallrye.certs</groupId>
+                <artifactId>smallrye-certificate-generator-junit5</artifactId>
+                <version>${smallrye-certificate-generator-junit5.version}</version>
```

Comment:
Could you please add a comment to remove this once https://github.com/quarkusio/quarkus/pull/43393 reaches us?

Reply:
I didn't bother because I'm planning to remove it immediately on `quarkus-main`.

Reply:
Good idea :)

Reply:
I added a comment as suggested in the latest commit.